content/en/docs/contributing/testing.md

```diff
@@ -28,4 +28,31 @@ $ git format-patch HEAD~5
 $ ./support/scripts/checkpatch.pl ./000*
 ```
 
+You can also run the `checkpatch` on one or multiple files before committing by using the `-f` option.
+Furthermore, some of the errors can be automatically solved by using the `--fix-inplace` option:
+
+```console
+$ ./support/scripts/checkpatch.pl --no-tree -f --fix-inplace <path/to/the/file>
+```
+
+For example, if you have some changes in the `lib/vfscore/vfs.h`, you can check and fix potential coding style issue with:
+
+```console
+$ ./support/scripts/checkpatch.pl --no-tree --fix-inplace -f lib/vfscore/vfs.h
+ERROR: "foo*bar" should be "foo *bar"
+#576: FILE: lib/vfscore/vfs.h:576:
++int	sys_chmod(const char*path, mode_t mode);
+
+total: 1 errors, 0 warnings, 828 lines checked
+...
+
+$ ./support/scripts/checkpatch.pl --no-tree --fix-inplace -f lib/vfscore/vfs.h
+total: 0 errors, 0 warnings, 828 lines checked
+
+lib/vfscore/vfs.h has no obvious style problems and is ready for submission.
+```
+
+Note that not all the coding style errors can be solved automatically and some of them should be addresed by hand.
```

Comment:
Use `alert` shortcodes for notes. See an example [here](https://github.com/unikraft/docs/blob/main/content/en/docs/contributing/docs.md?plain=1#L77-L79).